Automattic/simplenote-ios
# DO NOT MERGE

### Fix
This PR is to put together an installable build for my sanity to ensure that the changes in #1463 work

### Test
1. Install development build on iOS 12 device
2. Launch app
3. make sure no crash
